tests/php/View/SSViewerTest/TestGlobalProvider.php

```diff
@@ -43,4 +43,9 @@ public static function get_argmix()
         $args = func_get_args();
         return 'z' . implode(':', $args) . 'z';
     }
+
+    public static function getNull()
```

Comment:
I wanted to use `: null` as a typehint here but apparently PHP doesn't allow that 😅

Reply:
isset(null) === false && isnull(null) === true.
PHP doesn't have a concept of "is set, but is null", so I guess that makes sense 😅